apiary.apib

```diff
@@ -0,0 +1,167 @@
+FORMAT: 1A
+HOST: http://api.floworky.com/api/v1
+
+# floworky
+
+Floworky is a workflow tool for productivity 
+
+## Items Table [/item]
+
+### List All Items [GET]
+
+You want to list all the items. It takes the user id email from the parameters.
+
++ Request (application/json)
+
+    + Headers
+    
+            Location: /list/:user_id/:email
+            Authorization: JSONWEBTOKEN
+
++ Response 200 (application/json)
+    
+    + Body
+
+            {
+              "items": [
+                {
+                  "id": 1,
+                  "title": "give dev $200",
+                  "description": "egiftcard",
+                  "completed": false,
+                  "parent_id": 0,
+                  "user_id": 8,
+                  "createdAt": "2016-10-20T00:31:56.430Z",
+                  "updatedAt": "2016-10-20T00:31:56.430Z"
+                }
+              ],
+              "tree": [
+                {
+                  "id": 1,
+                  "parent_id": 0,
+                  "title": "give dev $200",
+                  "description": "egiftcard",
+                  "completed": false,
+                  "children": [
+                    {
+                      "id": 2,
+                      "parent_id": 1,
+                      "title": "check bank acct",
+                      "description": "",
+                      "completed": false,
+                      "children": []
+                    }
+                  ]
+                }
+              ]
+            }
+
+### Create a Item [POST]
+
+You want to create an item. It takes a JSON object containing the title of the item, the description and parent id and returns the id of the object and a success message.
```

Comment:
See notes above - describe data in response.  Let's see if we can find better formatting options for the parameters piece.